pkg/cmd/install.go

```diff
@@ -894,6 +898,16 @@ func getAddonsApplier(c *cli.Context, runtimeConfig *ecv1beta1.RuntimeConfigSpec
 	if adminConsolePwd != "" {
 		opts = append(opts, addons.WithAdminConsolePassword(adminConsolePwd))
 	}
+
+	if c.String("config-values") != "" {
+		err := configutils.ValidateKotsConfigValues(c.String("config-values"))
+		if err != nil {
+			return nil, fmt.Errorf("unable to validate config values file: %w", err)
```

Comment:
does this turn into the CLI output someone sees if they pass malformed config?

Reply:
yes